.docker/Dockerfile.ckan

```diff
@@ -6,7 +6,7 @@ ARG SITE_URL
 ENV SITE_URL="${SITE_URL}"
 
 ENV DOCKERIZE_VERSION v0.6.1
-RUN wget https://github.com/jwilder/dockerize/releases/download/${DOCKERIZE_VERSION}/dockerize-alpine-linux-amd64-${DOCKERIZE_VERSION}.tar.gz \
+RUN apk add --no-cache curl && curl -s -L -O http://github.com/jwilder/dockerize/releases/download/${DOCKERIZE_VERSION}/dockerize-alpine-linux-amd64-${DOCKERIZE_VERSION}.tar.gz \
```

Comment:
Why plain HTTP?

Reply:
can be made https again. was trying to get it working with alpine linux, in the end curl was better at working with proxy than wget.